clinica/pipelines/t1_freesurfer_longitudinal/t1_freesurfer_template_utils.py

```diff
@@ -1,1264 +1,214 @@
 # coding: utf8
-import clinica.utils.filemanip
-import clinica.utils.participant
 
-__author__ = "Alexis Guyot"
-__copyright__ = "Copyright 2016-2019, The Aramis Lab Team"
-__credits__ = ["Alexis Guyot"]
-__license__ = "See LICENSE.txt file"
-__version__ = "0.1.0"
-__maintainer__ = "Alexis Guyot"
-__email__ = "alexis.guyot@icm-institute.org"
-__status__ = "Development"
 
+def init_input_node(caps_dir, participant_id, list_session_ids, output_dir):
+    """Initialize the pipeline.
 
-def get_capsxsectional_path(caps_dir, subject, session):
-    """Get path to cross-sectional run in CAPS dir
+    This function will create folders and symbolic links in SUBJECTS_DIR env variable for upcoming run of recon-all.
 
-    Args:
-        caps_dir (string): location of CAPS folder
-        subject (string): participant ID
-        session_list (strings): session
+    Note(AG):
```

Comment:
Can you confirm that this info is also in the wiki of the pipeline?

Reply:
Not yet but I plan to add a warning/info the wiki page explaining the creation of folder in $TMP written in the CLI ([cf screenshot](https://user-images.githubusercontent.com/24808663/74757893-e633e280-5276-11ea-82b1-ce6a50287e27.png))

Reply:
For this situation, the command line looks like:
![T1FreeSurferTemplate](https://user-images.githubusercontent.com/24808663/75440927-fb86cc00-595c-11ea-8a6e-d94586baa106.png)

This is now explained on the Wiki:
<img width="617" alt="Wiki" src="https://user-images.githubusercontent.com/24808663/75439989-27a14d80-595b-11ea-8c53-5207b3c109f5.png">

:)